src/drivers/kinetis/tone_alarm/tone_alarm.cpp

```diff
@@ -440,17 +390,11 @@ void ToneAlarm::next_note()
 
 	if (updated) {
 		orb_copy(ORB_ID(tune_control), _tune_control_sub, &_tune);
-
-		if (_tunes.set_control(_tune) == 0) {
-			_play_tone = true;
-
-		} else {
-			_play_tone = false;
-		}
+		_play_tone = _tunes.set_control(_tune) == 0;
```

Comment:
:+1:

Reply:
This is now done the same in all architectures.